charts/kyverno/README.md

```diff
@@ -91,23 +91,23 @@ The following table lists the configurable parameters of the kyverno chart and t
 | `service.port`                     | port for the service                                                                                                                                                                                                                                         | `443`                                                                                                                                                                             |
 | `service.type`                     | type of service                                                                                                                                                                                                                                              | `ClusterIP`                                                                                                                                                                       |
 | `tolerations`                      | list of node taints to tolerate                                                                                                                                                                                                                              | `[]`                                                                                                                                                                              |
 | `securityContext`                  | security context configuration                                                                                                                                                                                                                               | `{}`                                                                                                                                                                              |
-| `podSecurityStandard`              | set desired pod security level `privileged`, `baseline`, `restricted`, `custom`. Set to `restricted` for maximum security for your cluster. See: https://kyverno.io/policies/pod-security/                                                                    | `baseline`                                                                                                                                                                         |
+| `podSecurityStandard`              | set desired pod security level `privileged`, `default`, `restricted`, `custom`. Set to `restricted` for maximum security for your cluster. See: https://kyverno.io/policies/pod-security/                                                                    | `baseline`                                                                                                                                                                         |
```

Comment:
Should it be "..`privileged`, **baseline**, `restricted`, `custom`... " ?

Reply:
And from that link page, it says "Pod Security Standard policies are organized in two groups, Baseline and Restricted". Where do we document level "privileged"?

Reply:
The whole chart needs to be updated to switch `default` to `baseline`, but yes that's another task. I think `privileged` needs to be removed as it basically meant "don't install any of these policies". It was an early definition of PSS but doesn't make sense to include. I'll fix the `privileged` statement, but then this PR is probably ok. Changing `default` to `baseline` is going to be a breaking change so maybe put that as a second issue mapping a PR? Is that a good idea?

Reply:
We already moved to `baseline` in [this](https://github.com/kyverno/kyverno/pull/1977) PR.  Is there anything left to do?

Yes we can fix `privileged` in a separate PR.

Reply:
@chipzoller - are you changing it back to `baseline`?